workers/unity/Assets/Playground/Scripts/MonoBehaviours/WorldCommandHelper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+
+public class WorldCommandHelper : IDisposable
```

Comment:
This is a lot of stuff to add. It's adding allocation, a disposable object, and a lot of code. Would it not be easier just to register callbacks separately as you do the same thing in each one and we give you the request object?

Reply:
That results in duplicate and less maintainable code wherever this class would be used, and was not easier to do that

Reply:
Surely it's exactly the same set of callbacks. Just registered one after the other instead of via a helper method.

Reply:
Is the duplicate more than having to put += after two require-able things?